config/schema/advanced_entity_reference_formatter.schema.yml

```diff
@@ -0,0 +1,17 @@
+field.formatter.settings.advanced_entity_reference_formatter:
+  type: field.formatter.settings.entity_reference_entity_view
+  label: 'Advanced entity reference formatter settings'
+  mapping:
+    override:
+      type: string
+      label: 'Override option'
+    bundle_view_mode:
+      type: sequence
+      label: 'Bundle View mode'
+      sequence:
+        type: string
+    view_mode_delta:
```

Comment:
Looks like we need a standard for settings naming cause if we have `view_mode_delta` then why is the other setting `bundle_view_mode` instead of `view_mode_bundle`.
Both are similar options: the prior sets view modes per delta and the last sets view modes per bundle.

Moreover, as mentioned on previous comment, this whole module works around the idea of view mode selection so it looks a bit redundant to constantly repeat `view_mode` in settings parameters.

Reply:
updated them to `bundle_modes` and `delta_modes`